Source/OrchBase/Placement.cpp

```diff
@@ -12,7 +12,7 @@ Placement::Placement(OrchBase * _p):iterator(0),par(_p),pCon(0),pD_graph(0)
 
 Placement::~Placement()
 {
-    if (iterator == 0) delete iterator;

```

Comment:
The aforementioned placement destructor mistake (you don't want to delete the iterator if it's not defined).